stories/features/welcome/page/screens/themeBox.tsx

```diff
@@ -32,13 +54,22 @@ export default class ThemingBox extends React.PureComponent<Props, {}> {
         <WelcomeThemeImage />
         <Title>{locale.chooseYourTheme}</Title>
         <Paragraph>{locale.findToolbarTheme}</Paragraph>
-          <PrimaryButton
-            level='primary'
-            type='accent'
-            size='large'
-            text={locale.theme}
-            onClick={onClick}
-          />
+        <SelectGrid>
+            <SelectBox onChange={this.onChangeThemeOption}>
+              <option value=''>{locale.selectTheme}</option>
+              <option value='Dark'>{locale.themeOption1}</option>
+              <option value='Light'>{locale.themeOption2}</option>
+              <option value='System'>{locale.themeOption3}</option>
```

Comment:
let's do:
- Light
- Dark
- System

Reply:
How about we bring in the strings from the product, and create an issue to change the existing ones to what you suggest @rossmoody ? Then it's consistent with Settings page either way?